openhcl/underhill_attestation/src/lib.rs

```diff
@@ -295,9 +295,12 @@ pub async fn initialize_platform_security(
             driver,
         )
         .await
-        .map_err(ErrorInner::RequestVmgsEncryptionKeys)?
+        .map_err(|e| {
+            tracing::error!("failed to retrieve key-encryption key");
```

Comment:
I wonder if this call really helpful. The message corresponding to `AttestationErrorInner::RequestVmgsEncryptionKeys(e)` is already logged at error level (appended to the final error) for this specific error path.
This call ends up printing the similar message at error level in the same error path.

Reply:
This is meant to add an error event to the span, which is separate from the error message.

Reply:
is there a span created for the loggings?

Reply:
A span is created when init_platform_security is called.

Reply:
I'm curious when error occurs whether this span will be closed with the error message already. 

https://github.com/microsoft/openvmm/blob/2e7ddc3e3d8bfcb0f7c428c2fd08b7c463fc0d00/openhcl/underhill_core/src/worker.rs#L1569